### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by martbetz** [Beep6581/RawTherapee#6601] [General]
I just wanted to thank everyone involved with RawTherapee for making such an indispensable tool accessable to everyone – personally, I have no idea what I'd do without it.

**Comment by ghost** [comment]
I would like to say the same.

I like the team behind!

RT is incredible good software 🥇

**Comment by rbyte** [comment]
I just discovered RawTherapee, and am blown away, too. This is great software, really. The best free Raw editor, as far as I can see, be far. A thousand thanks to all the developers and the masterminds behind this, and for contributing it to the free software universe! I bow to you in gratitude.
